minicargo.mk

```diff
@@ -239,7 +239,7 @@ LLVM_CMAKE_OPTS += CMAKE_BUILD_TYPE=Release
 LLVM_CMAKE_OPTS += $(LLVM_CMAKE_OPTS_EXTRA)
 
 
-$(LLVM_CONFIG): $(RUSTCSRC)build/Makefile
```

Comment:
`LLVM_CONFIG` is defined as:
```
LLVM_CONFIG := $(RUSTCSRC)build/bin/llvm-config
```

=> this isn't enough, isn't it?

Reply:
No, because I'm overriding LLVM_CONFIG on the command line. This rule does not specify how to build an arbitrary LLVM_CONFIG, it specifies how to build $(RUSTCSRC)/build/bin/llvm-config. When I specify some system LLVM I want this rule to not be triggered.